### ISSUE TITLE
Omit mappings and snapshot from generated SDs.

### ISSUE THREAD

**Issue by Healthedata1** [FHIR/sushi#186]
At least make an option for the generated SD to be just the diff and Metadata and no inherited mappings from the base spec.  I would prefer this to be a default as this is just visual noise for the editor.

**Comment by Healthedata1** [comment]
This is a direct representation of what the profile author is creating as well.

**Comment by cmoesel** [comment]
We actually have a task in this sprint to look at moving to only a diff.  TBH, I quickly tried it last week and the IG Publisher choked on something (that worked w/ the snapshot in place), which is why we decided to make it a full story in this sprint; some investigation will be needed.

As for mappings, maybe if we don't include the snapshot, the IG Publisher will take care of it; but the last time I checked, if we remove the top-level mappings from the SD, then the publisher emits errors because many of the individual element mappings now refer to non-existent top-level mappings.  But if we remove element mappings from the snapshot, then we're supposed to create differentials for them -- so it makes a real mess.  Hopefully if we remove the snapshot, then that gets around some of this.
